plugins/uniques.js

```diff
@@ -510,45 +733,149 @@ window.plugin.uniques.onMissionLoaded = function(data) {
 	});
 };
+/****************************************************************************************/
+/** Im-/Export of uniques ***************************************************************/
+/****************************************************************************************/
+window.plugin.uniques.optExport = function() {
+	var data = localStorage['plugin-uniques-data'];
+	window.saveFile(data, 'IITC-uniques.json', 'application/json');
+}
+window.plugin.uniques.optImport = function() {
+	L.FileListLoader.loadFiles({accept:'application/json'})
+		.on('load',function (e) {
+			try {
+				var msgExit = 'Please only confirm this if you know what you are doing!\n'
+				                    + 'Are you sure you want to save your Unique visits/captures back to IITC?';
+				if (!confirm(msgExit)) {
+				  return;
+				}
+				var data = JSON.parse(e.reader.result);
+
+				if (Object.keys(data).length) {
+				  window.plugin.uniques.uniques = data;
+				}
+				console.log('UNIQUES: reset and imported uniques.');
+				confirm('Import Successful.');
+				// to write back the data to localStorage
+				window.plugin.uniques.storeLocal('uniques');
+			} catch(e) {
+				console.warn('UNIQUES: failed to import data: '+e);
+			}
+		});
+}
+
+/*****************************************************************************************/
+/** UNIQUES Backlog to be processed whenever a portal's LatLonE6 can be resolved to GUID */
+/*****************************************************************************************/
+window.plugin.uniques.onPortalAdded = function(data) {
+	var guid = data.portal.options.guid;
+	var p = data.portal.options.data;
+	var id = p.latE6 + "," + p.lngE6;
+
+	//console.log("portal added: %s %s %o", guid, id, data);
+	// check if portal is in missedLatLngs
+	if (window.plugin.uniques.missedLatLngs[id]) {
+		console.log("found portal guid for previously missed portal: %s -> %s (%o)", id, guid, data);
+		window.plugin.uniques.uniques[guid] = Object.assign (
+			{},
+			window.plugin.uniques.uniques[guid],
+			window.plugin.uniques.missedLatLngs[id].action
+		);
+		window.plugin.uniques.storeLocal('uniques');
+		delete window.plugin.uniques.missedLatLngs[id];
+		window.plugin.uniques.storeLocal('missedLatLngs');
+		$("#missedPL").html(window.plugin.uniques.genList());
+	}
+}
+
+window.plugin.uniques.removeOldParsedMsgs = function() {
+//	remove all timestamps older than 30 days
+	var old = (Date.now() - (30*24*60*60*1000)); //30 days in miliseconds
+	var count = 0;
+	for (var item in window.plugin.uniques.parsedMsgs){
+		if (item < old) {
+			delete item;
+			count++;
+		}
+	}
+	console.log ('[uniques] removed %s old parsedMsgs', count);
+}
+
+window.plugin.uniques.genList = function (){
+	var mLL = window.plugin.uniques.missedLatLngs;
+	var list = '';
+	for (var item in mLL) {
+		var p = mLL[item].portal;
+		list = list + '<a onclick=\"map.setView(['+p.latE6/1E6+','+p.lngE6/1E6+'],15);\">'+p.name+'</a><br>';
```

Comment:
```suggestion
		list = list + '<a onclick="map.setView(['+p.latE6/1E6+','+p.lngE6/1E6+'],15);">'+p.name+'</a><br>';
```
And is it possible to replace inline js with jQuery listeners?

Reply:
changed